code/src/App.js

```diff
@@ -1,9 +1,38 @@
-import React from 'react';
+/* eslint-disable no-underscore-dangle */
+import React, { useState, useEffect } from 'react';
+import { ThoughtsOutput } from 'components/ThoughsOutput';
+import { ThoughtsInput } from 'components/ThoughtsInput';
 
 export const App = () => {
+  const [thoughtList, setThoughtList] = useState([])
+  const [loading, setLoading] = useState(false)
+  const submitHandler = (message) => {
+    if (message.length >= 5 || message.length <= 140) {
+      fetch('https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts', {
+        method: 'POST',
+        body: JSON.stringify({ message }),
+        headers: { 'Content-Type': 'application/json' }
+      }).then((response) => response.json()).then((data) => {
+        setThoughtList((pv) => [data, ...pv])
+      })
+    } else {
+      alert('make sure the input is between 5-140 characters long')
```

Comment:
This alert is not working for me if I do less than 5 characters or more than 140 it goes to a blank page and has errors in the console log that mention App.js line 11 and ThoughtsOutput.js:line 8 and 15:16.

Reply:
Thank you , did not notice this, sorted now!